public/components/trace_analytics/components/traces/traces.tsx

```diff
@@ -22,7 +21,6 @@ export interface TracesProps extends TraceAnalyticsComponentDeps {
 export function Traces(props: TracesProps) {
   return (
     <>
-      <DataSourcePicker modes={props.modes} selectedMode={props.mode} setMode={props.setMode!} />
```

Comment:
do we need datasource picker here?

Reply:
<img width="910" alt="Screenshot 2024-08-19 at 2 18 47 PM" src="https://github.com/user-attachments/assets/7e2ae0e5-2d9e-4d8f-92f5-a31caa8dca8c">
Moving the datasourcepicker to be inline with the search bar required it to be put inside <TraceContent>